app/use_cases/accounts/apps/chatwoots/webhooks/import_message.rb

```diff
@@ -1,20 +1,24 @@
 class Accounts::Apps::Chatwoots::Webhooks::ImportMessage
+  require 'open-uri'
+
   def self.call(chatwoot, contact, webhook)
     message = get_or_import_message(chatwoot, contact, webhook)
     { ok: message }
   end
 
   def self.get_or_import_message(chatwoot, contact, webhook)
     message = contact.events.where(
-      '? <@ additional_attributes', { chatwoot_id: webhook['id'] }.to_json
+      '? <@ additional_attributes', { chatwoot_id: webhook['conversation']['messages'].first['id'] }.to_json
```

Comment:
```suggestion
      ''? <@ additional_attributes', { chatwoot_id: webhook['id'] }.to_json
```

Aqui a forma antiga não seria melhor?

Reply:
Eu fiz essa alteração para a gnt n ter dúvidas se ele está pegando o id correto, pois a gnt recebe um webhook com vários ids, e teve um caso  q eu reparei q estava puxando o id errado de um outro caso.

Reply:
Entendi
Mas esse caso que você pegou com erro será que não pode ter sido nesse arquivo? `app/use_cases/accounts/apps/chatwoots/webhooks/process_webhook.rb`